src/cr-logging/Logging.cs

```diff
@@ -0,0 +1,108 @@
+﻿// <copyright file="Logging.cs" company="Cognisant">
+// Copyright (c) Cognisant. All rights reserved.
+// </copyright>
+
+namespace CR.Logging
+{
+    using System;
+    using System.Configuration;
+    using Microsoft.Extensions.Configuration;
+    using Serilog;
+    using Serilog.Core;
+    using Serilog.Events;
+    using Serilog.Formatting;
+
+    /// <summary>
+    /// Main Logging class. Holds the methods for setting up a SeriLog Logger.
+    /// </summary>
+    public static class Logging
+    {
+        private static IConfigurationRoot _appConfig;
```

Comment:
This isn't a static variable, it would be better to have this passed into the places that use it. We only care about this variable when running `SetupLogger`, nothing uses it once that method has complete.